.github/workflows/style.yaml

```diff
@@ -0,0 +1,53 @@
+name: Style
+on:
+  pull_request:
+    types:
+    - opened
+    - synchronize
+    - labeled
+
+env:
+  ROX_PRODUCT_BRANDING: RHACS_BRANDING
+
+jobs:
+
+  style-check:
+    if: |
+      (github.event.action == 'labeled' && github.event.label.name == 'turbo-build') ||
+      (github.event.action != 'labeled' && contains(github.event.pull_request.labels.*.name, 'turbo-build'))
+    runs-on: ubuntu-latest
+    container:
+      image: quay.io/stackrox-io/apollo-ci:stackrox-test-0.3.50
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
+        with:
+          fetch-depth: 0
+          ref: ${{ github.event.pull_request.head.sha }}
+
+      - name: Cache Go dependencies
+        uses: ./.github/actions/cache-go-dependencies
+
+      - uses: ./.github/actions/create-concatenated-ui-monorepo-lock
+
+      - name: Cache UI dependencies
+        uses: ./.github/actions/cache-ui-dependencies
+
+      - name: Cache QA Test dependencies
+        uses: ./.github/actions/cache-gradle-dependencies
+
+      - name: PR labels
+        uses: joerick/pr-labels-action@v1.0.6
+
+      - name: Setup Go build environment
+        run: |
+          if [[ -n "$GITHUB_PR_LABEL_CI_RELEASE_BUILD" ]]; then
+            cci-export GOTAGS release
```

Comment:
Does `cci-export` work in GHA?

Reply:
It's in our test container
```
docker run -it quay.io/stackrox-io/apollo-ci:stackrox-test-0.3.54 /bin/bash -c cci-export
Usage: /bin/real-bash KEY VALUE
```

Reply:
Ah, I got it. It's there but it's not working correctly :) Good catch

Reply:
Well, GOTAGS thing is gone now, but it was done before. See e.g. this revision:
https://github.com/stackrox/stackrox/blob/ba9956b42f9b90169c2ee047a7c1723ea3eb5377/.circleci/config.yml#L2249-L2255

I wonder if had any effect or was just there for consistency?

Reply:
Reply from @janisz in DM:

> https://github.com/stackrox/stackrox/blob/master/Makefile#L137-L142 golangci runs for two tags It think other checks are not tags sensitive